### ISSUE TITLE
Poses are camera-to-world?

### ISSUE THREAD

**Issue by FreemanG** [Kai-46/nerfplusplus#41]
According to your data description: 

> Poses are camera-to-world, not world-to-camera transformations.

Then why does [normalize_cam_dict.py](https://github.com/Kai-46/nerfplusplus/blob/master/colmap_runner/normalize_cam_dict.py) save W2C in the cam_dict? Shouldn't it be C2W (cam to world)?

**Comment by Kai-46** [comment]
You are right! You could get C2W easily by taking the inverse of W2C, and vice versa.

**Comment by FreemanG** [comment]
So I have to change this [line](https://github.com/Kai-46/nerfplusplus/blob/ebf2f3e75fd6c5dfc8c9d0b533800daaf17bd95f/colmap_runner/normalize_cam_dict.py#L55) 

> return np.linalg.inv(C2W)

to
  
> return C2W

Is this right? The experiments seem better so far.

**Comment by Kai-46** [comment]
Yes, or you could do such matrix inverses while converting the json camera file to text files.
